regular-expressions/introduction/basic-syntax-i/regex-wildcards-i.md

```diff
@@ -48,7 +48,7 @@ And would match any word character in a mixed string:
 
 > 💡 Using `\W` does the opposite of `\w` and would match all *non-word characters*.
 
-### `/\W*/`
+### `/\W/`
 
 '**!@£$%^&**' ✅
```

Comment:
do we want to match the entire string (`!@£$%^&`)? if so, we need to use `/\W*/` or `/\W+/`

if not, and we want to match each individual character in the string (`!`, `@`, ...) then we have to use `/\W/`